src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/ProjectSystemProviderContext.cs

```diff
@@ -12,15 +13,41 @@ namespace NuGet.PackageManagement.VisualStudio
     /// </summary>
     public class ProjectSystemProviderContext
     {
+        public const string RESTORE_PROJECT_STYLE = "RestoreProjectStyle";
+        public const string TARGET_FRAMEWORK = "TargetFramework";
+        public const string TARGET_FRAMEWORKS = "TargetFrameworks";
```

Comment:
nit: ``public static readonly``

we've been trying to avoid public const since that can't be changed easily (just a coding standard thing really)